framework/messages/de/yii.php

```diff
@@ -56,32 +56,32 @@
     'The requested view "{name}" was not found.' => 'Die View-Datei "{name}" konnte nicht gefunden werden.',
     'The verification code is incorrect.' => 'Der Prüfcode ist falsch.',
     'Total <b>{count, number}</b> {count, plural, one{item} other{items}}.' => 'Insgesamt <b>{count, number}</b> {count, plural, one{Eintrag} other{Einträge}}.',
-    'Unable to verify your data submission.' => 'Es ist nicht möglich, Ihre Dateneingabe zu prüfen.',
+    'Unable to verify your data submission.' => 'Dateneingabe konnte nicht überprüft werden oder ist ungültig.',
```

Comment:
only `Dateneingabe konnte nicht überprüft werden.`, there's nothing about verification in the original msg

Reply:
Given the context of the message, the new form is better because this message happens in two situations:

- CSRF token is not avaialble: "Dateneingabe konnte nicht überprüft werden"
- CSRF token is invalid:  "Dateneingabe ist ungültig"

Anyway I'd add 'Ihre' back:

```suggestion
    'Unable to verify your data submission.' => 'Ihre Dateneingabe konnte nicht überprüft werden oder ist ungültig.',
```

Reply:
> Given the context of the message, the new form is better because this message happens in two situations:
>
>CSRF token is not avaialble: "Dateneingabe konnte nicht überprüft werden"
>CSRF token is invalid: "Dateneingabe ist ungültig"

Shouldn't also the source message changed then?

Reply:
I've committed this suggestion, but @schmunk42, what do you think the source message should be changed to?

Reply:
I'd use `Ihre Dateneingabe konnte nicht überprüft werden.`